buildscripts/set_tensorflow_bazelrc.sh

```diff
@@ -58,3 +58,17 @@ build --color=yes
 build --verbose_failures
 build --spawn_strategy=standalone
 EOF
+echo "MKL flag: $MKL_FEATURE"
+
+if [ ! "${MKL_FEATURE+x}" ]; then
+# MKL if unset
+cat >> $BAZEL_RC_DIR/tensorflow.bazelrc << EOF
+build --define tensorflow_mkldnn_contraction_kernel=0
```

Comment:
I don't think there is a need to ensure eigen contraction kernels if not building MKL elsewhere.
`tensorflow_mkldnn_contraction_kernel` is enabled by default and is used in just about all x86 builds, unlike the broader mkl configure.

Reply:
This is actually to disable MKL stuff completely which is enabled by default. This was raised in the open-ce-community thread.